CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.18...3.24 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.19...3.24 FATAL_ERROR)
```

Comment:
This needs to be documented
- in the docs: https://vast.io/docs/setup-vast/build
- in a changelog entry

After that, feel free to check the reviewer checklist.